packages/providers/providers.yaml

```diff
@@ -1106,6 +1119,15 @@ booking-com:
             example: secure-supply|supply
             suffix: -xml.booking.com
             prefix: https://
+    credentials:
+        username:
+            type: string
+            title: Machine Account Username
```

Comment:
Machine account? Might be clearer with just Booking.com Username no?

Reply:
Thanks for the review, think this is fine